get.sh

```diff
@@ -245,13 +245,14 @@ getTestKitGenAndFunctionalTestMaterial()
 		OPENJ9_BRANCH="-b $OPENJ9_BRANCH"
 	fi
 
-	echo "git clone $OPENJ9_BRANCH $OPENJ9_REPO"
-	git clone -q $OPENJ9_BRANCH $OPENJ9_REPO
+	echo "git clone --depth 1 $OPENJ9_BRANCH $OPENJ9_REPO"
+	git clone --depth 1 -q $OPENJ9_BRANCH $OPENJ9_REPO
 
 	if [ "$OPENJ9_SHA" != "" ]
 	then
 		echo "update to openj9 sha: $OPENJ9_SHA"
 		cd openj9
+		git fetch --unshallow
 		git fetch -q --tags $OPENJ9_REPO +refs/pull/*:refs/remotes/origin/pr/*
 		git checkout -q $OPENJ9_SHA
```

Comment:
Is there a reason why we have `-q` on all the git commands?

Reply:
Seems like a question for @llxia. The code was introduced here:
https://github.com/AdoptOpenJDK/openjdk-tests/pull/404/files#diff-bb552f35560c477dfc858910b86984f0R120

Reply:
I think the intention was to reduce the unnecessary console output.